src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -0,0 +1,195 @@
+// TO DO: Replace made up properties and content with real versions once finalized.
+export const STATUS_TYPES = {
```

Comment:
Seems like EVENT_TYPES is the new version of this. Let's standardize on the other one and delete this one?

Reply:
Also...maybe we throw it into a new `constants` file? 🤷‍♂️

Reply:
Per our "phone" conversation, we'll check with @cmgiven to make sure the current status types map directly to the past event types first. If so, we can address it in another little PR.

Reply:
Responded in Slack, but note that there will be two different sets of types for events and statuses.